examples/typescript/vercel-ai-chatbot/lib/chat/actions.tsx

```diff
@@ -58,7 +58,7 @@ async function submitUserMessage(content: string) {
   let textNode: undefined | React.ReactNode
 
   const result = await streamUI({
-    model: openai('gpt-3.5-turbo'),
+    model: openai('gpt-4o-mini'),
```

Comment:
@ac3xx cool did not know we had gpt-3.5 here

can you use gpt-4o instead?

also we wanted to allow user to use ollama or chrome ai eventually too if you ever want to contribute to this?

https://github.com/mediar-ai/screenpipe/issues/55